client/mobile/utils/environment.js

```diff
@@ -5,4 +5,4 @@ var environments = {
   production: {server:'http://45.55.134.204', port:3000}
 }
 
-module.exports = environments.production;
+module.exports = environments.devDeployed;
```

Comment:
Can you reset this to production so everyone doesnt need to reset each time they pull? When you put in a pull request, either change this value back, or just commit all files except for this one. You can use git stash to help with this